src/index.html

```diff
@@ -4,11 +4,226 @@
+        </div>
+        <div class="w3-third">
+        <h3>NYC School of Marketing and Advertising</h3>
+        <h3>2007 - 2010</h3>
+        <p>This is a description of your education. Concisely describe your certifiate, degree or course in a way that’s easy for readers to scan quickly.</p>
+        </div>
+        <div class="w3-third">
+        <h3>L.A. University of Applied Sciences</h3>
+        <h3>2002 - 2006</h3>
+        <p>This is a description of your education. Concisely describe your certifiate, degree or course in a way that’s easy for readers to scan quickly.</p>
+        </div>
+        </div>
+      </div>
+      <div class="item">
+        <h2>HOBBIES & INTERESTS</h2>
+        <div class="row">
+        <div class="col"><img width="150" height="150" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSf71rBnViRBrals8RJieIrnGbQj8kNO8X-8cbRNKKDDnaQauOH" /></div>
+        <div class="col"><img width="150" height="150" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQE-dsatETF-ekn9rAlcePwgiHrj-BeCQ0S0hQ-6mSVfppwxoAKqQ" /></div>
+        <div class="col"><img width="150" height="150" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQuSJkBoSGrQPcPc_BcB2s91aKj8qStfFhmLYJvUUC17p_kMYT4Lg" /></div>
+        <div class="col"><img width="150" height="150" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQV6T06UjO9aO_oRuItOlST-3vje_Wk8SYIjiyrI1KipUZQZDGVtw" /></div> 
+        <div class="col"><img width="150" height="150" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQYdbr-D9yjgBm0UEoB__IXe3VwPnxt9PLQelrX31rv3gC6Ok-oIw" /></div>
+        <div class="col"><img width="150" height="150" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQhPdmqbxJdRSrtI579c2_FwK0xrY2PLCXF0bnKjUg_fr2cAUwzRQ" /></div>
+        </div>
+     </div>
+       
+     </div>
+     </div>
+     
+     <h2>Skills</h2>
+     
+     <h2>QUICK</h2>
+     <table>
+     <tr>
+     <td>Phone</td>
+     <td>123-456-7890</td>
+     </tr>
+     <tr>
+     <td>Email</td>
+     <td>info@mysite.com</td>
+     </tr>
+     <tr>
+     <td>Website</td>
+     <td>www.wix.com</td>
+     </tr>
+     <tr>
+     <td>Address</td>
+     <td>500 Terry Francois Street San Francisco, CA 94158</td>
+     </tr>
+     </table>
+     <h2>CONTACT ME</h2>
+     <form action="" method="post" >
+     <label for="user">Name:</lable>
+     <input type="text" id=user name="user" value="" />
+     <br>
+     <label for="email">Email:</lable>
+     <input type="text" id=email name="email" value="" />
+     <br>
+     <label for="subjuct">Subjuct:</lable>
+     <input type="text" id=subjuct name="subjuct" value="" />
+     <br>
+     <label for="message">Massage:</lable>
+     <textarea name="textarea" value="" rows="6" cols="25" ></textarea>
+     <br>
+     <input type="button" value="Submit">
+     </form>
+
+     <button >DOWNLOAD MY FULL RESUME</button>
```

Comment:
`<button>`

Reply:
What is means <button>?

Reply:
plz remove the extra empty space :P

Reply:
It seems that no-biggie, but a good programmer always care about tiny details :)

Reply:
haha, OK